### PR TITLE
Release DPL-554, Y24-107

### PR DESCRIPTION
**Pull request by KatyTaylor**
#### Changes proposed in this pull request

[DPL-554](https://github.com/sanger/sequencescape/issues/3696) SS New tag sets for Chromium Library Plate Manifest template:
This adds a new tag set will be added to Chromium Library Plate Manifest: PN-1000212 Single Index Kit N, Set A
 
[Y24-107](https://github.com/sanger/sequencescape/issues/4122) Sequencescape search function fails to handle exception
This resolves errors while searching a short search string with All option in Sequencescape.

[Y24-061](https://github.com/sanger/sequencescape/issues/4085) Research the best way to pass Pool XP tube export messages from Limber to RabbitMQ 

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-18.0.0
\ No newline at end of file
+22.3.0
```
Comment (sdjmchattie) [inline .nvmrc:1]: Could we put a new line at the end of this file?

Comment (KatyTaylor) [reply .nvmrc:1]: Sure, what's the reason?

**Comment by sdjmchattie** [comment]
This also includes https://github.com/sanger/sequencescape/issues/4085 which hasn't been fully tested yet.

File: .nvmrc (hunk 1 above)
Comment (sdjmchattie) [reply .nvmrc:1]: According to the definition of a line in the POSIX standards https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206 it must end with a new line character.  That's why GitHub marks it with a red circle at the end of the file.  Technically this is a file with no lines in it.  I don't think it breaks NVM's reading of the file, but many tools don't handle this well.

**Comment by KatyTaylor** [comment]
> This also includes #4085 which hasn't been fully tested yet.

Yep, is that OK? The code won't be called yet will it? (Although technically could be if anyone had the endpoint)

**Comment by sdjmchattie** [comment]
> > This also includes #4085 which hasn't been fully tested yet.
> 
> Yep, is that OK? The code won't be called yet will it? (Although technically could be if anyone had the endpoint)

I'm not too worried.  I just wanted to point it out.  It's currently in UAT without any of the config it needs and it's not causing issues.

**Review by sdjmchattie — APPROVED** [review]

**Pull request merged**